### DISCUSSION TITLE
Install Strapi in Cloudways (managed hosting) - Cannot access admin

### DISCUSSION THREAD

**Discussion by ventouris** [strapi/strapi#6026] [Legacy Topics]
**Describe the bug**
Cloudways is a hosting provider which gives a limited access to ssh (no sudo). However, I have been able to installed other node.js apps without a problem. Cloudways uses both Apache and nginx. I tried a lot of suggestions from Google and Slack, but I am out of ideas now.

**Steps to reproduce the behavior**

1. Run `npx create-strapi-app .` to install strapi in current directory
2. Add database credentials (done without error. Tables are being created in the MySQL)
3. Update `npm run build` (no error)
4. Update server.js based on documentation. Port 1337 is closed, so I had to use another open port (3000).

```
{
  "host": "localhost",
  "port": 3000,
  "proxy": {
    "enabled": true,
    "ssl": true,
    "host": "mantis.stackprime.com",
    "port": 443
  },
  "cron": {
    "enabled": false
  },
  "admin": {
    "autoOpen": false
  }
}
```
5. Finally, run `npm run develop` where I get the following message

```
┌─────────────────────────────────────┐
│ https://mantis.stackprime.com/admin │
└─────────────────────────────────────┘

[2020-05-01T15:06:54.979Z] info ⏳ Opening the admin panel...
```
The root url returns 403 Forbidden and the /admin returns 404 Not Found. 

If I add a plain test.html file in the root and open it with /test.html, I can see the content.

Also, if I try to open the url /build, the browser tab changed to "Strapi Administrator", page stays blank and I get the error in browser console `https://mantis.stackprime.com/admin/runtime~main.230c20d9.js 404 not found`

**Expected behavior**
I expected to open the admin and create the admin account. 

**System**

- Node.js version: v14.1.0
- NPM version: 6.14.4
- Strapi version: 3.0.0-beta.20.1
- Database: mySQL 5.7
- Operating system: Debian GNU/Linux 8.11 (jessie)

In the same server, I was able to install a node.js app with a server.js file as

```
const express = require('express');
const bodyParser = require('body-parser');

const app = express();
const port = 65300;

app.use(express.json());
require('./app/routes')(app, {});

app.listen(port, () => {  console.log('We are live on ' + port);});
```

And this in .htaccess

```
RewriteEngine On
RewriteBase /
RewriteRule ^(.*)?$ http://127.0.0.1:65300/$1 [P,L]
```

However, it doesn't work with Strapi.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by ventouris** [comment]
The only reason I put it as a bug was that I followed documentation and it didn’t work. Plus another node.js app in the same server is working. So I thought it might be in the strapi side. Either on documentation or config files. But of course if it is suited more as discussion I appreciate it :)

**Reply by derrickmehaffy** [reply]
A few things have changed since this was last updated:

```
Node.js version: v14.1.0
NPM version: 6.14.4
Strapi version: 3.0.0-beta.20.1
```

Strapi doesn't support node v14 (yet) and only v10 or v12 is supported
Currently as of writing the beta is also no longer supported, only the stable release.

I'm going to mark this question as answered but should you still have the same problem in stable, please do reply back and we can try to troubleshoot further.
